tests/fuzzing/corpus/init_fuzzer_seed1

```diff
@@ -0,0 +1,130 @@
+&{
```

Comment:
How is this seed file useful? Doesn't seem to be unmarshallable.

Reply:
Think of the seed as a file that helps the fuzzer get started with mutating relevant data. Instead of starting from scratch, we provide a seed so that the fuzzer can mutate this instead.
In my local testing the seed helped greatly with generating relevant input data.